compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolResolver.java

```diff
@@ -1322,4 +1324,48 @@ private boolean isMemberAllowed(SymbolEnv env, BSymbol symbol) {
                 && env.enclInvokable.symbol.receiverSymbol.type.tsymbol == symbol.owner
                 || isMemberAllowed(env.enclEnv, symbol));
     }
+
+    public void loadFunctionalConstructors() {
+        BPackageSymbol xmlModuleSymbol = symTable.langXmlModuleSymbol;
+        if (xmlModuleSymbol == null) {
+            return;
+        }
+
+        ScopeEntry entry = xmlModuleSymbol.scope.lookup(names.fromString("Element"));
+        while (entry.symbol != null && entry.symbol != symTable.notFoundSymbol) {
+            if (entry.symbol.tag == SymTag.CONSTRUCTOR) {
+                return;
+            }
+            entry = entry.next;
+        }
```

Comment:
Similar comment as above regarding this. Since this method seems to be called only once, will there be a case where the symbols are already defined?